examples/java-blob/README.md

```diff
@@ -0,0 +1,215 @@
+# Spring Boot + Azure Storage + AAD Pod Identity
+
+This tutorial is based on [this documentation](https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-storage).
+
+This repository contains sample code and instructions demonstrating how you can access Azure Storage using a user assigned manage identity with the aid of AAD Pod Identity from a spring boot application.
+
+This sample takes a `POST` request and saves in as a text file in an Azure Blob Storage container.
+
+## Prerequisites
+
+* AKS cluster
+* AAD Pod Identity installed and configured
+* Azure Container Registry
+* Azure Storage
+    * Make sure you have already created a container
+
+## Setup
+
+### Create Manage Identity and Assign Role
+
+In this step we'll create a new manage identity which will be used to interact with the Azure Storage Account.
+
+**Note:** this step can be skipped if you already have a manage identity you'd like to reuse.
+
+1. Setup:
+
+    ```sh
+    AKS_NAME=<YOUR_AKS_CLUSTER_NAME>
+    AKS_RESOURCEGROUP=$(az aks list --query "[?name == '$AKS_NAME'].resourceGroup" -o tsv)
+
+    MANAGE_IDENTITY_NAME=<YOUR_MANAGE_IDENTITY_NAME>
+    MANAGE_IDENTITY_RESOURCEGROUP=<YOUR_RESOURCE_GROUP>
+
+    STORAGE_ACCOUNT_NAME=<STORAGE_ACCOUNT_NAME>
+    STORAGE_ACCOUNT_RESOURCEGROUP=$(az storage account list --query "[?name == '$STORAGE_ACCOUNT_NAME'].resourceGroup" -o tsv)
+
+    AKS_MSI_CLIENTID=$(az aks show \
+        -n $AKS_NAME \
+        -g $AKS_RESOURCEGROUP \
+        --query "identityProfile.kubeletidentity.clientId" \
+        -o tsv)
+
+    STORAGE_ACCOUNT_RESOURCEID=$(az storage account show \
+        --name $STORAGE_ACCOUNT_NAME \
+        --query 'id' -o tsv)
+    ```
```

Comment:
Could you rename the environment variables to those used in https://azure.github.io/aad-pod-identity/docs/demo/standard_walkthrough/?

Reply:
No problem! Done.